extensions/wikia/WikiFactory/SpecialWikiFactory_ajax.php

```diff
@@ -565,21 +560,21 @@ function axWFactorySaveVariable() {
 				$return = Wikia::successmsg( "Parse OK, variable saved." );
 			break;
 			default:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this "switch/case" number of lines from 30 to at most 10, for example by extracting code into function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1151)